app/models/order.rb

```diff
@@ -5,6 +5,8 @@ class Order <ApplicationRecord
   has_many :items, through: :item_orders
   belongs_to :user
 
+  enum status: %w(Packaged Pending Shipped Cancelled)
```

Comment:
Good work with the enum! We'll need to make model tests for different statuses, but it looks great!